src/services/languageService.js

```
@@ -0,0 +1,42 @@
+
+    // Convert to array of objects
+    const languages = Object.entries(languagesMap).map(([code, name]) => ({
+      code,
+      name: name.trim() // Ensure clean strings
+    }));
+
+    // Sort by language name
+    return languages.sort((a, b) => a.name.localeCompare(b.name));
+    
+  } catch (error) {
+    console.error('Error fetching languages:', error);
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)